iron-spi-kafka/src/main/java/io/axway/iron/spi/kafka/KafkaTransactionStore.java

```diff
@@ -76,6 +77,8 @@ public void close() throws IOException {
 
     @Override
     public void seekTransactionPoll(BigInteger latestProcessedTransactionId) {
+        // This dummy poll registers the assignments
+        m_consumer.poll(100);
```

Comment:
can be moved at init time or in the store factory ?

Reply:
found a better way :)